bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -145,11 +188,11 @@
 	</channel-type>
 
 	<channel-type id="modulationLevel">
-		<item-type>Number</item-type>
+		<item-type>Number:DimensionLess</item-type>
```

Comment:
The state built for this channel is already a QuantityType ?
I saw no change in that PR for this channel.
What about thing migration ? This leads to the discussion we have for color temperature channels. If we change the item type, I guess that the channel must be removed and added through the migration tool. But it has to be verified.

Reply:
And if you change type, README has to be updated too.

Reply:
Fixed readme. 
About the migration i'm not sure. Does it also apply to dimensionless? I use file based configuration. But obviously i can test this.

Reply:
Tested and added upgrade instructions as indeed they are needed.
Minor difference that i performed an update and not add + remove. Checked the jsondb and the item type is updated.

Reply:
Fine